examples/events/EiffelIssueVerifiedEvent/simple.json

```diff
@@ -36,6 +36,18 @@
     {
       "type": "ISSUE",
       "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee5"
```

Comment:
Should remove link of type "ISSUE" from the example as it isn't a valid link type after changes

Reply:
Makes Sense, Done